x11-wm/muffin/muffin-4.8.0.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils gnome2-utils xdg-utils virtualx
```

Comment:
I didn't see `eutils` being used in here. And it shouldnt't have any ebuild usage left anymore, probably a remnant?